pkgs/tools/misc/beam-book/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib, stdenv
+, fetchFromGitHub
+, wget, asciidoctor, ditaa, graphviz
+, gitMinimal
+, erlang
+, jdk
+, rsync
+, rouge
+}:
+
+let
+  pname = "the-beam-book";
+  version = "0.0.14.fix";
+
+  bookDirName = "${pname}-${version}";
+
+  book = fetchFromGitHub {
+    name = bookDirName;
+    owner = "happi";
+    repo = "theBeamBook";
+    rev = version;
+    sha256 = "06r19063a9s6hg84garn83rn4cnnkvs58x3j68zvf94mwsmfs86h";
+    leaveDotGit = true;
+  };
+
+  erlangGenOp = fetchFromGitHub {
+    name = "erlang.tar.gz";
+    owner = "erlang";
+    repo = "otp";
+    rev = "master";
+    sha256 = "1qfdry8wdba3abfsa1qkkcglww186j8dyl40si1vrwsc7f221r1d";
+  };
+
+in stdenv.mkDerivation rec {
+  inherit pname version;
+
+  srcs = [ book erlangGenOp ];
+
+  outputs = [ "out" "pdf" ];
```

Comment:
```suggestion
  outputs = [ "out" "doc" ];
```

Reply:
I didn't want `doc` because this outputs a website (set of HTML files) and a PDF, but no other documentation format.

Reply:
The doc output is the right for this. Manpages are in man and html, pdf, rtf, etc goes to doc.

Reply:
In that case, everything should go to `doc` output then?